src/data.ts

```diff
@@ -63,7 +63,7 @@ export class Data {
    * @param   cityData : The array of DataPoint describing the city.
    * @returns Returns the average of the city's DataPoint values.
    */
-   static getMeanCityData(cityData: DataPoint[]): number {
+  static getMeanCityData(cityData: DataPoint[]): number {
     if (cityData.length <= 0) {
       return 0;
```

Comment:
Note that everything in data.ts was just linted, no real code changes took place despite the large gitdiff